hc/settings.py

```diff
@@ -15,10 +15,11 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 HOST = "localhost"
-SECRET_KEY = "---"
+SECRET_KEY = os.getenv("SECRET_KEY")
 DEBUG = True
 ALLOWED_HOSTS = [
     'healthchecks-akira.herokuapp.com',
+    '*'
 ]
 DEFAULT_FROM_EMAIL = 'healthchecks@example.org'
```

Comment:
It makes sense to remove this line now and use the value `healthchecks@example.org` as the default value when [`DEFAULT_FROM_EMAIL` isn't set in the environment rather than `None`](https://github.com/andela/healthchecks_akira/pull/15/files#diff-992aedc6fffb368eaee59c319dfa012eR131)